packages/maskbook/src/database/Plugin/wrap-plugin-database.ts

```diff
@@ -28,6 +28,11 @@ export function createPluginDatabase<Data extends IndexableTaggedUnion>(plugin_i
             if (!data) return undefined
             return data.value as Of<T>
         },
+        async has<T extends Type>(type: T, id: ID<T>): Promise<boolean> {
+            const t = await c('r')
+            const count = await t.store.count(key({ type, id }))
+            return count > 1
```

Comment:
i need add an has method in plugin database wrapper

CC @Jack-Works

Reply:
LGTM